go/vt/topo/server.go

```diff
@@ -67,30 +68,32 @@ const (
 
 // Filenames for all object types.
 const (
-	CellInfoFile             = "CellInfo"
-	CellsAliasFile           = "CellsAlias"
-	KeyspaceFile             = "Keyspace"
-	ShardFile                = "Shard"
-	VSchemaFile              = "VSchema"
-	ShardReplicationFile     = "ShardReplication"
-	TabletFile               = "Tablet"
-	SrvVSchemaFile           = "SrvVSchema"
-	SrvKeyspaceFile          = "SrvKeyspace"
-	RoutingRulesFile         = "RoutingRules"
-	ExternalClustersFile     = "ExternalClusters"
-	ShardRoutingRulesFile    = "ShardRoutingRules"
-	KeyspaceRoutingRulesFile = "KeyspaceRoutingRules"
+	CellInfoFile           = "CellInfo"
+	CellsAliasFile         = "CellsAlias"
+	KeyspaceFile           = "Keyspace"
+	ShardFile              = "Shard"
+	VSchemaFile            = "VSchema"
+	ShardReplicationFile   = "ShardReplication"
+	TabletFile             = "Tablet"
+	SrvVSchemaFile         = "SrvVSchema"
+	SrvKeyspaceFile        = "SrvKeyspace"
+	RoutingRulesFile       = "RoutingRules"
+	ExternalClustersFile   = "ExternalClusters"
+	ShardRoutingRulesFile  = "ShardRoutingRules"
+	CommonRoutingRulesFile = "Rules"
```

Comment:
It might still be worth each one having its own unique file name. But this is fine too IMO as when looking at the full path it makes sense.

Reply:
To be clear, I think it's fine as-is. I was just jotting down my thought FWIW — which may be nil 🙂. We can resolve this thread I think.